frontend/src/components/ChatBox/ChatBoxFeed.tsx

```diff
@@ -12,14 +13,21 @@ function ChatBoxFeed({ messages }: { messages: ChatMessage[] }) {
 	const chatboxFeedContainer = useRef<HTMLDivElement>(null);
 	const isOverflow = useIsOverflow(chatboxFeedContainer);
 
+	useEffect(() => {
+		if (chatboxFeedContainer.current) {
+			chatboxFeedContainer.current.scrollTop =
+				chatboxFeedContainer.current.scrollHeight;
+		}
+	}, [messages]);
+
 	return (
-		<div
+		<section
 			className="chat-box-feed"
 			ref={chatboxFeedContainer}
-			// eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
 			tabIndex={isOverflow ? 0 : undefined}
```

Comment:
Now that messages inside the feed are focusable, we don't need this anymore. So we can also remove the `/* eslint-disable jsx-a11y/no-noninteractive-tabindex */` at the top of this file.